src/main/resources/schemas/V60__smartdocuments_informatie_object_type.sql

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+ALTER TABLE ${schema}.smartdocuments_document_creation_template
+    ADD informatie_object_type_url VARCHAR NOT NULL;
```

Comment:
Maybe a bit too late but still: did we make a decision to use English for new additions to the ZAC data model? Because we will very likely not be the only users of this data model. Think of e.g. future BI usage by councils directly on our datamodel etc. Then again, it is our datamodel so if it makes our lives easier we should do it I guess. Still, good to have a shared decision on this.

Reply:
> Maybe a bit too late but still: did we make a decision to use English for new additions to the ZAC data model? Because we will very likely not be the only users of this data model. Think of e.g. future BI usage by councils directly on our datamodel etc. Then again, it is our datamodel so if it makes our lives easier we should do it I guess. Still, good to have a shared decision on this.

If this is the case, then perhaps we should stick to Dutch in columns/tables

Reply:
Let's stick to EN for new stuff for now. It's already a mixed bag anyway. :-)